### DISCUSSION TITLE
What's the relation with gqless?

### DISCUSSION THREAD

**Discussion by borekb** [gqty-dev/gqty#44] [Q&A] [answered]
I'm not very familiar with either but I've seen [on Twitter](https://twitter.com/Zephraph/status/1425553284056616967) that this is a "rebranded gqless" project (if I understand correctly?!).

Could someone explain:

- The relation of the projects
- Why the new name
- If some technical fundamentals are changing, for example, https://gqless.com/ says _"no more code generation"_ while https://gqty.dev/docs/getting-started shows codegen setup as one of the first things.

I'm not personally familiar with either of the projects but they are something we'd like to evaluate as a GraphQL client for us so I'd like to understand their similarities / differences and their relation in general.

**Comment by PabloSzx** [comment, marked as answer]
gqty is no more than continuing the development of gqless, I'm the one behind all the rewrite of gqless v2, but the original owner removed me as an admin, and right now he is completely disappeared, any difference in the docs or descriptions are no more than a result of the remake of the website, there is no extra meaning behind everything like if it was worth a conspiracy theory.

The name was basically just a result of brainstorming with @natew and it sounded good and it made sense, there isn't a very deep meaning behind it

**Reply by borekb** [reply]
Thanks. And about the technical differences, like gqless mentioning type inference instead of code generation while GQty seems to be based on codegen?

**Reply by PabloSzx** [reply]
no, both are the same, it's the same library but we are continuing its development here, that's it

**Reply by bkniffler** [reply]
@PabloSzx Appreciate all your work! Could you please add a note to the readme though? I think the situation is worth a notice. Its a bit hard to follow up and understand. "rebranded gqless" doesn't do justice IMHO, but the maintainer removing you as admin and disappearing, with no more maintenance in sight is a good reason to start a different project.
